### PR TITLE
feat: multiple accounts on beinleumi base

### PR DESCRIPTION
**Pull request by nissant**
Changes:
- Add graceful support for scrapping multiple accounts on beinleumi base -> fetchAccounts method. This is a 3 year old TODO item in the existing code base comments :) 
- I have a FIBI account and was able to test the scrapper successfully.

### CODE REVIEW

**Comment by baruchiro** [comment]
Hi @nissant, Thank you for your contribution!

I'm reviewing your suggestion. Meanwhile, if you think you can help us more, I'm inviting you to register as a Code Owner: #830

**Review by baruchiro — COMMENTED** [review]
Thank you for your contribution!

Usually, when you suggest a change, we ask for more contributors with the same bank account to test your change, because sometimes the website acts differently for different users (depending on user config and account...)

For this case, we need someone to test your change with Beinleumi or Massad, but no one was registered on #830

File: src/scrapers/base-beinleumi-group.ts [hunk 1]
```diff
@@ -281,15 +281,37 @@ async function fetchAccountData(page: Page, startDate: Moment) {
...
+  if (!accountsIds.length) {
+    const accountData = await fetchAccountData(page, startDate);
+    accounts.push(accountData);
+  } else {
+
+    for (const accountId of accountsIds) {
+      if (accountsIds.length > 1) {
+        await page.select('#account_num_select', accountId);
+        await waitUntilElementFound(page, '#account_num_select', true);
+      }
```
[the remark this answers is not in the document]
Comment (nissant) [reply src/scrapers/base-beinleumi-group.ts]: Thanks @baruchiro  it's much cleaner like that. Corrected.

**Review by baruchiro — APPROVED** [review]
Thank you!

**Comment by nissant** [comment]
@baruchiro Looks like all the checks have passed and PR is ready to merge, can you please allow it?

**Comment by baruchiro** [comment]
Thank you for the reminder

**Pull request merged**